### ISSUE TITLE
How to update?

### ISSUE THREAD

**Issue by splintering** [pal1000/Realtek-UAD-generic#177]
I installed 6.0.9718.1 on a more or less freshly installed Win11 - no previous Realtek drivers were present. Installation went smoothly, even the Realtek Audio Control was installed through the store (interestingly in the Asus version - does the store know about my mainboard and fetch a customized installer?).

Without any need I came here after that to see if there is an updated package. So I downloaded 6.0.9738.1 and ran the setup.cmd again. During step "Removing generic Realtek UAD components..." I got a repeating error that a driver cannot be removed because it is being used by a device.

I tried a lot of things to overcome that, including forceupdate, going into safe mode, deactivating the device on Windows level, and now even on BIOS level, but still the same result:

![grafik](https://github.com/user-attachments/assets/0ee2e4a6-6ec0-4ff9-8be5-01ecfae04ea0)

DriverStoreExplorer always shows the same three drivers that cannot be removed:

![grafik](https://github.com/user-attachments/assets/fcb5239c-7965-4c6f-81bf-0dd65dc56229)

forceupdater seems to fail with a file not found error:

![grafik](https://github.com/user-attachments/assets/989d2b00-7e4e-431c-9a6e-4d29c59e3f16)

(or maybe that is no issue, because after that the rest of the forceupdater runs thorugh without further errors).

I am not sure what the correct way would have been to update the package and now think I should not have done it at all because everything was running fine... Also if I reactivate the devices I do have sound and maybe no need to install the driver package after all, My main motivation was to have up-to-date drivers without the usual bloatware.

Any hints would be appreciated :-)

**Comment by pal1000** [comment]
Do you something attached that uses your  sound card?

> does the store know about my mainboard and fetch a customized installer?).

Realtek Audio console detects motherboard vendor from first 4 digits of SUBSYS value of your sound card hardware ID. It has special skins for ASUS and MSI at least.

**Comment by splintering** [comment]
> Do you something attached that uses your sound card?

No, I deliberately disconnected everything before I tried it the last time.
